tasks/fix-cat2.yml

```diff
@@ -1144,26 +1144,48 @@
       - notimplemented
 
 - name: "MEDIUM | RHEL-07-030320 | PATCH | The audit system must take appropriate action when the audit storage volume is full."
-  command: "true"
-  changed_when: no
+  block:
+    - name: "Install audispd-plugins"
+      yum:
+        name: audispd-plugins
+        state: present
+
+    - name: "Set disk_full_action"
+      lineinfile:
+        path: /etc/audisp/audisp-remote.conf
+        regexp: "disk_full_action =.*"
+        line: "disk_full_action = {{ rhel7stig_audisp_disk_full_action }}"
+    # Technically, network_failure_action should only be in 030321, but it's also called out in 030320.
+    - name: "Set network_failure_action"
```

Comment:
I think we should remove the network_failure_action task from this block. Technically it is in the 030320 STIG ID fix text but it seems like an error that should be fixed in a later STIG revision. Its just a duplicative task in this role now.